### PR TITLE
[corlib] Removed MVID and AOTID metadata.

### PR DESCRIPTION
**Pull request by esdrubal**
MVID and AOTID metadata is no longer displayed at the end of the stack
trace.

Now, every stack frame contains a MVID and AOTID inlined.

An IL offset stackframe looks like:
```  at StackTraceDumper.Main () [0x00000] in <a3ec873a838c42c29f4eba7cab7d1dfb>:0```

A native offset stack frame looks like:
```  at StackTraceDumper.<Main>m__0 () <0x100e15d50 + 0x00029> in <a3ec873a838c42c29f4eba7cab7d1dfb#b37a9d11247b811ffffdaff8090d8a59>:0```

### CODE REVIEW

File: mcs/class/corlib/System.Diagnostics/StackTrace.cs [hunk 1]
```diff
@@ -182,6 +182,15 @@ public virtual StackFrame[] GetFrames ()
 			return frames;
 		}
 
+		private static string aotid;
+		private static string GetAotId ()
```
Comment (marek-safar) [inline mcs/class/corlib/System.Diagnostics/StackTrace.cs]: No need for private noise

File: mcs/class/corlib/System.Diagnostics/StackTrace.cs [hunk 2]
```diff
@@ -182,6 +182,15 @@ public virtual StackFrame[] GetFrames ()
...
+		{
+			if (aotid == null)
+				aotid = new Guid (Assembly.GetAotId ()).ToString ("N");
```
Comment (marek-safar) [inline mcs/class/corlib/System.Diagnostics/StackTrace.cs]: This is failing one of tests due to GetAotId being null

File: mcs/tools/mono-symbolicate/StackFrameData.cs [hunk 3]
```diff
@@ -106,11 +97,9 @@ internal void SetLocation (string file, int lineNumber)
...
 		}
 
-		public override string ToString () {
-			if (Line.Contains ("<filename unknown>:0") && LineNumber != -1)
-				return Line.Replace ("<filename unknown>:0", string.Format ("{0}:{1}", File, LineNumber));
-
-			return Line;
+		public override string ToString ()
+		{
+			return string.Format ("{0} in {1}:{2} ", Line.Substring (0, Line.IndexOf(" in <")), File, LineNumber);
```
Comment (marek-safar) [inline mcs/tools/mono-symbolicate/StackFrameData.cs]: Missing StringComparison in IndexOf

**Comment by esdrubal** [comment]
build!

**Comment by esdrubal** [comment]
build

**Pull request merged**